src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,86 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table;
+    private double threshold;
+    private int size;
+
+    public MyHashMap() {
+        this.table = new Node[DEFAULT_CAPACITY];
+        this.threshold = DEFAULT_LOAD_FACTOR * DEFAULT_CAPACITY;
+        size = 0;
+    }
 
     @Override
     public void put(K key, V value) {
-
+        int nodeIndex = getIndex(key);
+        if (size == threshold) {
+            resize();
+        }
+        Node<K, V> node = findNode(key, nodeIndex);
```

Comment:
Индекс нужно подсчитывать внутри findNode(). Для подсчета индекса тебе нужен только key, и он есть внутри findNode(). Всегда нужно стремиться к минимальному количеству аргументов в методе.
```suggestion
        Node<K, V> node = findNode(key);
```